cmd/kubelet/app/server.go

```diff
@@ -271,7 +271,7 @@ func initKubeletConfigSync(s *options.KubeletServer) (*componentconfig.KubeletCo
 // not be generated.
 func Run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) error {
 	if err := run(s, kubeDeps); err != nil {
-		return fmt.Errorf("failed to run Kubelet: %v", err)
+		return fmt.Errorf("failed to run Kubelet: %v\n", err)
```

Comment:
The issue here is that when this error is reported, and then kubelet restarted, the next error created will be on the same line causing a lot of confusion. I suppose we could start the logs with /n at the beginning but that seems a bit odd to me.